src/m365/spo/commands/list/list-get.ts

```diff
@@ -53,8 +55,10 @@ class SpoListGetCommand extends SpoCommand {
 
     const propertiesSelect: string = args.options.properties ? `?$select=${encodeURIComponent(args.options.properties)}` : ``;
 
+    const permissionsExpand: string = args.options.withPermissions ? `${args.options.properties ? '&' : '?'}$expand=HasUniqueRoleAssignments,RoleAssignments/Member,RoleAssignments/RoleDefinitionBindings` : ``;
```

Comment:
Wouldn't it be better to turn `propertiesSelect` into a `let` and assign the value to it instead of introducing a new const?